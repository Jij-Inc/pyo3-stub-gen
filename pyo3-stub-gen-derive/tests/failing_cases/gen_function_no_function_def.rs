use pyo3_stub_gen_derive::gen_function_from_python;

fn main() {
    pyo3_stub_gen::inventory::submit! {
        gen_function_from_python! {
            r#"
import typing

# Only imports, no function definition
            "#
        }
    }
}
