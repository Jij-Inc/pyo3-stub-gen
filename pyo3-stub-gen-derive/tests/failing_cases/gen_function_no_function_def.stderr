error: No function definition found in Python stub
  --> tests/failing_cases/gen_function_no_function_def.rs:6:13
   |
 6 | /             r#"
 7 | | import typing
 8 | |
 9 | | # Only imports, no function definition
10 | |             "#
   | |______________^
